### PR TITLE
TESTING - DO NOT MERGE

### PR DESCRIPTION
**Pull request by ranvijayj**
## JIRA Ticket Number

JIRA TICKET: ADO-1234

## Description of change
(REMOVE ME) Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Checklist (OPTIONAL):

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: experimental/views.py [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+    'v2/date/': date_v2,
+    'v2/time/': time_v2,
+    'v2/number/': number_v2,
+    'v2/phone_number/': phone_number_v2,
+    'v2/number_range/': number_range_v2,
+}
+
+
+def to_bool(value):
+    # type: (Union[str, unicode, bool, int, None]) -> bool
```
Comment (haptik-deployment) [inline experimental/views.py:149]: F821 undefined name 'Union'
F821 undefined name 'unicode'

File: experimental/views.py [hunk 2]
```diff
@@ -0,0 +1,232 @@
+import concurrent.futures
```
Comment (haptik-deployment) [inline experimental/views.py:1]: W1618 [34mimport missing `from __future__ import absolute_import`[0m

File: chatbot_ner/urls.py [hunk 3]
```diff
@@ -35,6 +36,8 @@
     url(r'^v2/phone_number/$', api_v2.phone_number),
     url(r'^v2/number_range/$', api_v2.number_range),
 
+    url(r'^experimental/detect/', detect_entities),
+
     # V2 bulk detectors
```
Comment (ranvijayj) [inline chatbot_ner/urls.py:41]:
```suggestion
    # V2 Bulk detectors
```

File: requirements.txt [hunk 4]
```diff
@@ -25,3 +25,4 @@ flake8==3.4.1
 pyaml==19.4.1
 coverage==4.5.3
 nose-exclude==0.5.0
+futures==3.3.0
```
Comment (ranvijayj) [inline requirements.txt:28]: futures==3.3.0

# Test comment Remove

Comment (ranvijayj) [inline requirements.txt:28]:
```suggestion
futures==3.3.0
# Test 123
```

**Pull request closed without merging**